lib/rules/no-deprecated-classes.js

```diff
@@ -23,7 +23,19 @@ const replacements = new Map([
   ['no-wrap', 'text-no-wrap'],
   ['ellipsis', 'text-truncate'],
   ['left', 'float-left'],
-  ['right', 'float-right']
+  ['right', 'float-right'],
+  ['display-4', 'text-h1'],
```

Comment:
This should not replace 2.3 classes if someone is on the vuetify 2.0-2.2

Reply:
Oh, I see... I'll think solution for this.

Reply:
I rebased to master branch and added new rule for replacing typography classes. 
How do you think it?

Reply:
tbh i think that the best solution would be detecting what version of vuetify is installed